commitizen/cmd.py

```diff
@@ -20,4 +20,10 @@ def run(cmd: str) -> Command:
     )
     stdout, stderr = process.communicate()
     return_code = process.returncode
-    return Command(stdout.decode(), stderr.decode(), stdout, stderr, return_code)
+    return Command(
+        stdout.decode("iso-8859-1"),
```

Comment:
Could you elaborate why do we need to decode `iso-8859-1`? Thanks!

Reply:
I changed to iso-8859-1 because my gpg's stdout returns latin characters that fails to be decoded.

The character I get into stdout is "à" .

Probably the best way could be doing multiple decode stages in case of exception, like "first decode with utf, if it fails decode with iso-8859, if it fails decode with... " , but it is a different task/issue than this PR.

Reply:
on windows I've just checked, it depends on the shell:

with Powershell I get nothing
![image](https://user-images.githubusercontent.com/842026/179483717-5e8483d9-3eff-40f9-9d9d-9fab1ff41284.png)

Meanwhile with MINGW I get something
![image](https://user-images.githubusercontent.com/842026/179483777-63aa28ad-b8e6-4904-8d45-2abd553ffde2.png)

this is why I currently use chardet when I've to decode something, and I'm using it on windows and on linux.

Reply:
latest commits group add chardet and its stub for mypy .

Reply:
I'm good with using chardet. Looks like a good idea.